src/content/docs/infrastructure/host-integrations/host-integrations-list/unix-monitoring-integration.mdx

```diff
@@ -14,8 +14,155 @@ freshnessValidatedDate: never
 
 The Unix monitoring integration allows for system-level monitoring of Unix-based servers, such as AIX, Linux, macOS, and Solaris/SunOS, and reports data to New Relic.
```

Comment:
Can we say a bit about what this monitoring lets someone do? Why someone would want to do this?